src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -29,9 +29,32 @@ import { TERMINAL_COMMAND_ID } from 'vs/workbench/parts/terminal/common/terminal
 import { Command } from 'vs/editor/browser/editorExtensions';
 import { timeout } from 'vs/base/common/async';
 import { FindReplaceState } from 'vs/editor/contrib/find/findState';
+import { execSync } from 'child_process';
+import { isWindows } from 'vs/base/common/platform';
 
 export const TERMINAL_PICKER_PREFIX = 'term ';
 
+function getCwd(configHelper: ITerminalConfigHelper, activeInstance: ITerminalInstance): string {
```

Comment:
This could be a handy utility function to have on `ITerminalInstance`

Reply:
Since it requires a terminal instance anyway I made it a member 😄